cclib/method/nuclear.py

```diff
@@ -9,12 +9,39 @@
 
 import logging
 
-import numpy
+import numpy as np
 
 from cclib.method.calculationmethod import Method
 from cclib.parser.utils import PeriodicTable
 
 
+def get_most_abundant_isotope(element):
+    """Given a `periodictable` element, return the most abundant
```

Comment:
I'm not a fan of strictly keeping to the 80 column limit. Having slightly longer lines so that they don't wrap in a n ugly way is fine IMO, as long as the length is reasonable. What do you think? We could also cover this in #555.

Reply:
I think I'm in agreement; I have a habit of running `fill-paragraph` (or your editor's equivalent, `gq`?) for docstrings, and the default column width is 70. I can stop doing that.

Reply:
Yeah, let's have a convention to use only one tool if we want to auto-format. We have enough pep8 violations that it would make sense to format in bulk once if we want to go in that direction.